sunpy/map/mapbase.py

```diff
@@ -1695,24 +1679,19 @@ def plot(self, annotate=True, axes=None, title=True,
         >>> aia.draw_grid()   # doctest: +SKIP
 
         """
-
-        # extract hiddden kwarg
-        _basic_plot = imshow_kwargs.pop("_basic_plot", False)
-
         # Get current axes
         if not axes:
             axes = wcsaxes_compat.gca_wcs(self.wcs)
 
-        if not _basic_plot:
-            if not wcsaxes_compat.is_wcsaxes(axes):
-                warnings.warn("WCSAxes not being used as the axes object for this plot."
-                              " Plots may have unexpected behaviour. To fix this pass "
-                              "'projection=map' when creating the axes",
+        if not wcsaxes_compat.is_wcsaxes(axes):
+            warnings.warn("WCSAxes not being used as the axes object for this plot."
+                          " Plots may have unexpected behaviour. To fix this pass "
+                          "'projection=map' when creating the axes",
+                          SunpyUserWarning)
+            # Check if the image is properly oriented
+            if not np.array_equal(self.rotation_matrix, np.identity(2)):
+                warnings.warn("The axes of this map are not aligned to the pixel grid. Plot axes may be incorrect.",
```

Comment:
Stupid question. Were these changes not in a different PR?

Reply:
yes, but they were inside a `if basic_plot` block, they no longer are.

Reply:
OH I'M JUST BLIND AND DIDN'T NOTICE, CARRY ON.